eng/Dependencies.props

```diff
@@ -58,6 +58,8 @@ and are generated based on the last package release.
   <ItemGroup Label="External dependencies" Condition="'$(DotNetBuildFromSource)' != 'true'">
     <LatestPackageReference Include="BenchmarkDotNet" Version="0.10.13" />
     <LatestPackageReference Include="Microsoft.AspNet.WebApi.Client" Version="5.2.6" />
+    <LatestPackageReference Include="Azure.Security.KeyVault.Secrets" Version="4.0.0" />
+    <LatestPackageReference Include="Azure.Identity" Version="1.0.0" />
     <LatestPackageReference Include="Microsoft.Azure.KeyVault" Version="2.3.2" />
```

Comment:
That's for the old package which we aren't doing anything with in this change.

Reply:
Yeah, this file lists all dependencies for packages within the entire repo and since the old packages are still present and supported, we need to keep that around.